src/skadi/sample.cc

```diff
@@ -241,11 +200,155 @@ template <class coord_t> double sample::get(const coord_t& coord) const {
   return value / adjust;
 }
 
-template <class coords_t> std::vector<double> sample::get_all(const coords_t& coords) const {
+::valhalla::skadi::sample::sample(const std::string& data_source) : data_source(data_source) {
```

Comment:
i know it was like this before but do you mind removing the namespacing here since its not needed